### PR TITLE
Extend docker from crops yocto image

### PR DESCRIPTION
**Pull request by chuanjin**
Signed-off-by: Chuan Jin <cjin@luxoft.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,18 +9,21 @@ RUN mkdir /var/run/sshd
...
     
 RUN rm -rf /var/lib/apt/lists/*
     
 # en_US.utf8 is required by Yocto sanity check
 RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 ENV LANG en_US.utf8
 
+RUN userdel -r yoctouser
 RUN useradd --uid $userid vagrant --create-home --user-group --groups sudo
 RUN echo 'vagrant:vagrant' | chpasswd
```
Comment (tobsan) [inline Dockerfile]: Is it possible to tell vagrant to use a different user than "vagrant"? In that case it would be nicer to be able to still use "yoctouser"

File: Vagrantfile [hunk 2]
```diff
@@ -35,7 +32,7 @@ Vagrant.configure(2) do |config|
     # Configure username and password in git
     config.vm.provision "shell", privileged: false, path: "vagrant-cookbook/system-config/vagrant-ssh-user.sh"
 
-    # Configure username and password in git
+    # Download and set up the repo tool
```
Comment (tobsan) [inline Vagrantfile]: This should go into a separate commit imho.

File: Dockerfile (hunk 1 above)
Comment (chuanjin) [reply Dockerfile]: Yes, then I replace vagrant with yoctouser

**Comment by chuanjin** [comment]
Same build error, maybe disable cache?

**Comment by tobsan** [comment]
This build error is really weird since it seems the Jenkinsfile doesn't pick up on the changed variable content. We tried removing all docker images and their vagrant contexts but still same result. @rpannek do you have any idea what it could be?

**Comment by rpannek** [comment]
Just for the record since we talked about it in between, because @chuanjin is not in the Pelagicore organization he isn't allowed to run a job with a changed Jenkinsfile.
